.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## What is the goal of this pull request?
+
+## What does this pull request change?
+
+## What work remains to be done?
+
+## Does this pull request change the functionality of the code?
```

Comment:
I think we can deduce from the `what does this pull request change` question along with the review itself what functionality is changing.  Let's remove this question and the subsequent `how` question.